.github/workflows/build-test.yaml

```diff
@@ -19,8 +20,6 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Install upstream libseccomp
-        uses: ./.github/actions/setup
```

Comment:
Is this unnecessary in the main branch as well?
In that case, it would be better to fix the main at first, then let's do cherry-pick in order to make commits of `release/v0.3.x` branch same as that of the main as much as possible.

Reply:
@rusty-snake Do you have any update?:)

Reply:
Originally I wanted to test if this works. Then I - wanted to - add it back because I'm not sure if this has an influence on `build.rs` scripts.